packages/download-dialog/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@converse-plugins/download-dialog",
+  "version": "0.0.1",
+  "keywords": [
+    "converse.js",
+    "plugin"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
```

Comment:
I wanted to mention that you'll need to set this, but I see you've done so already.